### PR TITLE
Feature/backstage sql spl chars

### PR DESCRIPTION
**Pull request by hmuthusamy**
*Issue #, if available:*

*Description of changes:*
Removed special characters on the backstage pwd

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: platform/infra/terraform/mgmt/terraform/backstage.tf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 resource "random_password" "backstage_postgres_password" {
   length           = 48
```
Comment (shapirov103) [inline platform/infra/terraform/mgmt/terraform/backstage.tf]: length is crazy to me as well. I see some folks had issues when it was just truncated. Let's make it 16 or something like that

Comment (hmuthusamy) [reply platform/infra/terraform/mgmt/terraform/backstage.tf]: Fixed it to 16

Comment (elamaran11) [reply platform/infra/terraform/mgmt/terraform/backstage.tf]: I agree

Comment (elamaran11) [reply platform/infra/terraform/mgmt/terraform/backstage.tf]: Lets test it

**Review by elamaran11 — APPROVED** [review]

**Pull request merged**